desktop/cmp/grid/GridModel.js

```diff
@@ -34,7 +34,7 @@ export class GridModel {
     @observable.ref columns = [];
     @observable.ref sortBy = [];
     @observable groupBy = null;
-
+    @observable agNodesRendered = false;
```

Comment:
Need some kind of doc someplace to indicate what this is all about. It could be a comment on this property or an addition to the top-level class comment - I would favor the property.

Do we anticipate that applications will need to hook into this as part of the model's public API? If so, trying to think of a more general name so we don't necessarily bake "ag" into the model API...  Would `nodesRendered` / `rowsRendered` / `dataRendered` be an improvement?  Or `xxxRenderComplete` for any of those options?

Was also thinking something with `ready` in it to abstract a bit from the notion of "rendering" being required for certain operations, but ready seems more like a state change that happens once...

Reply:
Yes, number of critical questions here.  I was actually thinking we might hold such implementation specific state in the component itself (rather than the model).  And hoping that this whole thing could be invisible to applications.  

I don't think its the case that our models have to hold *all* state.  I think the key is that they hold the state that we expect applications, and external users of the component to read/write.

In this case the GridModel is the central API object for applications.  Would be happy to leave this particular wart off of it, if applications don't need it.

Note that 'agAPI' made more sense to put on the model, as I assumed it would be a useful 'escape hatch/entry point' for applications to the broad world of the ag grid API.